### DISCUSSION TITLE
Оповещение о записи разговора

### DISCUSSION THREAD

**Discussion by humingos** [mikopbx/Core#60] [Ideas (Идеи)]
Было бы отлично, если бы система позволяла уведомлять в начале разговора о записи разговора.
Вижу это так.

В системе есть возможность указать какой-то звуковой файл, который будет воспроизводиться в начале каждого разговора. Есть галочка, которая регулирует, надо ли воспроизводить файл при внутренних звонках.

Воспроизведение начинается только и сразу после начала разговора, т.е. после того, как отвечающая сторона подняла трубку. Воспроизводимый файл должен быть слышен обоим сторонам. В идеале воспроизведение должно идти прям в сам разговор, т.е. по сути разговаривать можно уже на фоне воспроизводимого файла. Воспроизводимый файл должен попадать и в запись разговора.

**Comment by boffart** [comment]
Подобные планы есть. Мб. опишем блог на эту тему. 
Основаная проблема нехватка времени, потому по срокам сориентировать не могу.

**Comment by boffart** [comment]
См. статью
[Оповещение о записи разговора](https://wiki.mikopbx.com/faq:call_recording_alert)

**Reply by jorikfon** [reply]
Я думаю в минимальном функционале надо такое реализовать, в некоторых странах незаконно использовать запись разговора без оповещения, а у нас нет такой возможности в принципе.
Я бы в общих настройках добавил возможность указать звуковой файл, если он указан то засунуть его во все диалпланы при соединении.
Вроде не самая большая задача, добавлю ее в список на будущие релизы.

**Comment by jorikfon** [comment]
Реализовано в релизе  2021.2.194

**Reply by humingos** [reply]
Хотел поблагодарить Вас за добавление этой фичи в сборку по умолчанию.

Очень приятно, что вы прислушиваетесь к своим пользователям. Так держать и большое спасибо!
